open_academy/views/views.xml

```diff
@@ -0,0 +1,61 @@
+<odoo>
+  <data>
+    <!-- explicit list view definition -->
+<!--
```

Comment:
Same here. Don't add commented code. Please remove this file and all the references to it.